packages/react-adminer/src/components/FavoriteList.tsx

```diff
@@ -35,29 +34,49 @@ const FavoriteList: React.FC<Props> = inputEntityName => {
 					setLocalStorageData(finalObjContent);
 				} else {
 					setIsEmpty(true);
+					setLocalStorageData(undefined);
 				}
 				return;
 			} catch {
 				//
 			}
 		}
 		setIsEmpty(true);
-	}, []);
+	}, [reducerValue]);
+
+	const deleteFavoriteFilter = (record: any): void => {
```

Comment:
Zkusme nepsat any

Reply:
Změněno na: (record: Record<string, number>)

Reply:
A je to opravdu number? Dává to smysl? Není tam náhodou url - string?